app/furniture/marketplace/orders/index.html.erb

```diff
@@ -2,5 +2,5 @@
 <%- @pagy, @records = pagy(orders.paid) %>
 <div class="mx-auto w-full max-w-prose">
   <%= render @records %>
-  <%== pagy_nav(@pagy) %>
+  <%== pagy_nav(@pagy) if @pagy.pages > 1 %>
```

Comment:
I'm not sure there's an reason to show pagination for a single page?

Reply:
Agree, I noticed that in another spot; makes me think we need a partial or Component for rendering the Pagy nav so we follow the same rule of skipping when no page